testsIT/src/test/scala/com/stratio/crossdata/driver/DriverIT.scala

```diff
@@ -75,7 +74,8 @@ class DriverIT extends EndToEndTest {
 
   "Crossdata Driver" should "be able to close the connection and start it again" in {
     assumeCrossdataUpAndRunning
-    var driver = Driver.getOrCreate(); Driver.getOrCreate()
+    var driver = Driver.getOrCreate();
```

Comment:
The test  check  that the driver can be reused, closing and opening again, so I think that it is necessary to be a var. But that problem it's not related with this PR